clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/callbacks/Utils.groovy

```diff
@@ -36,6 +36,26 @@ class Utils {
     }
   }
 
+  /**
+   * Return a single port string if a port range refers to a single port (e.g. 80-80).
+   *
+   * @param portRange - Port range to parse.
+   * @return - Single port if the ports in the port range are the same.
+   */
+  static String getPortFromPortRange(String portRange) {
```

Comment:
This might be more explicitly named something like `derivePortOrPortRangeDescription`.